### ISSUE TITLE
add support for remote files

### ISSUE THREAD

**Issue by sgratzl** [sgratzl/slack_cleaner2#6]
see https://api.slack.com/methods/files.remote.list

**Comment by sgratzl** [comment]
blocked by https://github.com/os/slacker/issues/160

**Comment by sgratzl** [comment]
as an alternative the corresponding API could be directly called using the slacker wrapper

**Comment by sgratzl** [comment]
wontfix
